### PR TITLE
JBDS 9.0.0.GA blog

### PR DESCRIPTION
**Pull request by alexeykazakov**
Do not publish yet!

### CODE REVIEW

**Comment by alexeykazakov** [comment]
@maxandersen @nickboldt please review

File: blog/ga-for-mars.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2015-10-06
+
+http://tools.jboss.org/downloads/jbosstools/mars/4.3.0.Final.html[JBoss Tools 4.3] and http://tools.jboss.org/downloads/devstudio/mars/9.0.0.GA.html[Red Hat JBoss Developer Studio 9] for Eclipse Mars are now generally available!
+
+image::/blog/images/jbds9.png[]
+
+CAUTION: Java 8 is required for installing and using of JBoss Tools. We still support developing and running applications using older Java runtimes. See more in link:/blog/2015-06-23-beta1-for-mars.html#java-8-to-run-eclipse-older-runtimes-ok-for-builds-deployment[Beta1 blog].
```
Comment (nickboldt) [inline blog/ga-for-mars.adoc]: "installation and use" or "installing and using"... don't need the "of" in there.

File: blog/ga-for-mars.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+JBoss Developer Studio comes with everything pre-bundled in its installer. Simply download it from our https://www.jboss.org/products/devstudio.html[JBoss Products page] and run it like this:
+ 
+    java -jar jboss-devstudio-<installername>.jar
+
+JBoss Tools or Bring-Your-Own-Eclipse (BYOE) JBoss Developer Studio require a bit more:
+
+This release requires at least Eclipse 4.5 (Mars) but we recommend
+using the latest http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/marsr[Eclipse 4.5.1 Mars JEE Bundle] since then you get most of the dependencies preinstalled. 
+
+Once you have installed Eclipse, you can either find us on the Eclipse Marketplace under "JBoss Tools" or "JBoss Developer Studio".
```
Comment (nickboldt) [inline blog/ga-for-mars.adoc]: ... or "Red Hat JBoss Developer Studio".

File: blog/ga-for-mars.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+For JBoss Tools, you can also use our update site directly if you are up for it.
```
Comment (nickboldt) [inline blog/ga-for-mars.adoc]: "if you are up for it?" Makes it sound challenging. It's not. :|

**Comment by maxandersen** [comment]
content wise looks good for me.

**Comment by alexeykazakov** [comment]
Thanks guys. I have updated the PR.

**Comment by alexeykazakov** [comment]
Pushed to master and production

**Pull request closed without merging**